cmd/wave/main.go

```diff
@@ -255,6 +271,26 @@ func main() {
 	wave.Run(conf)
 }
 
+// getEmptyOIDCValues checks if any of the required OIDC values were not set and returns which were not.
+func getEmptyOIDCValues(auth wave.AuthConf, requiredEnvOIDC map[string]string) []string {
+	var emptyRequiredOIDCParams []string
+	for key, val := range requiredEnvOIDC {
+		if val == "" {
+			switch key {
+			case "oidc-client-id":
+				emptyRequiredOIDCParams = append(emptyRequiredOIDCParams, "oidc-client-id")
+			case "oidc-client-secret":
+				emptyRequiredOIDCParams = append(emptyRequiredOIDCParams, "oidc-client-secret")
+			case "oidc-provider-url":
+				emptyRequiredOIDCParams = append(emptyRequiredOIDCParams, "oidc-provider-url")
+			case "oidc-redirect-url":
+				emptyRequiredOIDCParams = append(emptyRequiredOIDCParams, "oidc-redirect-url")
+			}
+		}
+	}
```

Comment:
When hinting the hashmap, I hoped for something like:
```suggestion
	for param, val := range requiredEnvOIDC {
		if val == "" {
			emptyRequiredOIDCParams = append(emptyRequiredOIDCParams, param)
		}
	}
```

Reply:
Ah yes my bad, not sure how I didn't realize that.